packages/renderer/src/docs/git.vue

```diff
@@ -1,3 +1,262 @@
+            <div class="row">
+              <div class="col">
+                <button class="q-btn q-btn--rectangle bg-secondary text-white">
+                  <div class="col items-center justify-center row"><i class="q-icon on-left material-icons"
+                      role="img">check_circle</i>Commit</div>
+                </button>
+              </div>
+              <div class="col">
+                <q-item-section>Once ready, click "commit" to store your changes locally.</q-item-section>
+              </div>
+            </div>
+          </q-item-section>
+        </q-item>
+
+        <q-item>
+          <q-item-section>
+            <div class="row">
+              <div class="col">
+                <button class="q-btn q-btn--rectangle bg-secondary text-white">
+                  <div class="col items-center justify-center row"><i class="q-icon on-left material-icons"
+                      role="img">change_circle</i>Reset</div>
+                </button>
+              </div>
+              <div class="col">
+                <q-item-section>Click "RESET" to undo your latest changes and convert the ARC to the last saved
+                  commit.</q-item-section>
+              </div>
+            </div>
+          </q-item-section>
+        </q-item>
+
+      </q-list>
+    </div>
+
+    <q-separator inset />
+
+    <div class="q-item row">
+      <div class="q-item__section column q-item__section--side justify-center">
+        <i class="q-icon text-grey-8 material-icons">wifi_protected_setup</i>
+      </div>
+      <div class="q-item__section column justify-center q-item__label">Push / Pull</div>
+    </div>
+
+    <div class="q-pt-sm text-justify">
+      <q-list>
+
+        <q-item>
+          <q-item-section>
+            <div class="row">
+              <div class="col">
+                <div class="q-checkbox row items-center q-checkbox--dense">
+                  <div class="q-checkbox__inner relative-position non-selectable q-checkbox__inner--falsy"><input
+                      class="hidden q-checkbox__native absolute q-ma-none q-pa-none" type="checkbox">
+                    <div class="q-checkbox__bg absolute"><svg class="q-checkbox__svg fit absolute-full"
+                        viewBox="0 0 24 24">
+                        <path class="q-checkbox__truthy" fill="none" d="M1.73,12.91 8.1,19.28 22.79,4.59"></path>
+                        <path class="q-checkbox__indet" d="M4,14H20V10H4"></path>
+                      </svg></div>
+                  </div><span class="no-outline"></span>
+                  <div class="q-checkbox__label">Use Large File <br> Storage</div>
+                </div>
+              </div>
+              <div class="col">
+                <q-item-section>Check this box to use <a
+                    href="https://www.nfdi4plants.org/nfdi4plants.knowledgebase/docs/guides/arc_WorkingWithLargeDataFiles.html"
+                    target="_blank">large file storage (LFS)</a></q-item-section>
```

Comment:
This link does not open for me on windows.

Reply:
I guess it must be whitelisted:

![image](https://github.com/nfdi4plants/ARCitect/assets/39732517/c53cefbc-950b-4094-bb53-b82f832b32de)

Reply:
done